sdk/textanalytics/ai-text-analytics/src/generated/models/index.ts

```diff
@@ -176,11 +186,15 @@ export interface DocumentSentiment {
   /**
    * Document level sentiment confidence scores between 0 and 1 for each sentiment class.
    */
-  documentScores: SentimentConfidenceScores;
+  confidenceScores: SentimentConfidenceScores;
```

Comment:
You should also include this change in the changelog

Reply:
This type only exists internally. Its properties get folded into the `AnalyzeSentimentResult`, except this property gets renamed, so this change doesn't manifest in the API surface.